src/caffe/proto/caffe.proto

```diff
@@ -393,8 +393,10 @@ message LayerParameter {
   optional ReductionParameter reduction_param = 136;
   optional ReLUParameter relu_param = 123;
   optional ReshapeParameter reshape_param = 133;
+  optional ROIPoolingParameter roi_pooling_param = 146;
```

Comment:
146 is now taken by recurrent_param in the latest master. Also, be sure to update the "last added parameter" at the top of the structure.